### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by binary-sequence**
- Leap 15 is now the current release
- Reorganize steps to make configuration after installation more understandable
- In Leap 15 we don't have anymore the issue with sqlite, this section can be moved back to advanced configuration

### CODE REVIEW

File: docs/Installing.asciidoc [hunk 1]
```diff
@@ -64,22 +64,21 @@ To add the development repository to your system, you can use these commands.
 zypper ar -f obs://devel:openQA/openSUSE_Tumbleweed devel-openQA
 
 
-# openSUSE Leap 42.3
-zypper ar -f obs://devel:openQA/openSUSE_Leap_42.3 devel-openQA
-zypper ar -f obs://devel:openQA:Leap:42.3/openSUSE_Leap_42.3 devel-openQA-perl-modules
+# openSUSE Leap 15 (newest)
```
Comment (foursixnine) [inline docs/Installing.asciidoc]: I don't think specifying older or newest (newer) would fit here :)

File: docs/Installing.asciidoc [hunk 2]
```diff
@@ -246,68 +265,49 @@ installed 'os-autoinst' from packages make sure to pass +--isotovideo+ option
...
-[openid]
-## base url for openid provider
-provider = https://www.opensuse.org/openid/user/
-## enforce redirect back to https
-httpsonly = 1
---------------------------------------------------------------------------------
-
-OpenQA supports only OpenID version up to 2.0. Newer OpenID-Connect and OAuth is
-not supported currently.
+[[database]]
```
Comment (foursixnine) [inline docs/Installing.asciidoc]: Database configuration is not part of advanced configuration, please move it back :) (and keep it before Authentication)

Comment (binary-sequence) [reply docs/Installing.asciidoc]: I don't understand why it is basic configuration.

I consider basic configuration something required for the _thing_ to work. All the other steps are required to configure even with default values, the database not.

Comment (foursixnine) [reply docs/Installing.asciidoc]: @binary-sequence If you don't set up the database, you won't be able to run openQA unless you install the openqa-local-db package, which is very specific to openSUSE.

**Pull request merged**